### PR TITLE
fix docker build env

### PR DESCRIPTION
**Pull request by zzpmiracle**

### CODE REVIEW

File: .github/workflows/reusable.yml [hunk 1]
```diff
@@ -72,7 +72,6 @@ jobs:
         docker build -t disc-dev-${{ inputs.name }} \
           --build-arg BASEIMAGE=${{ inputs.base_image }} \
           ${{ inputs.extra_build_args }} \
-          ${{ inputs.extra_envs }} \
```
Comment (qiuxiafei) [inline .github/workflows/reusable.yml:75]: why removed..

Comment (tanyokwok) [reply .github/workflows/reusable.yml:75]: +1

Comment (zzpmiracle) [reply .github/workflows/reusable.yml:75]: `docker build` does not have  `-e` option

**Review by tanyokwok — APPROVED** [review]
LGTM

**Pull request merged**